src/applications/simple-forms/21-0845/tests/e2e/0845-auth-disclose.cypress.spec.js

```diff
@@ -71,7 +27,6 @@ const testConfig = createTestConfig(
           }).click();
         });
       },
-      ...pageTestConfigs,
```

Comment:
note: 
This is all tested automatically anyway, so removing is basically the same. 
The only part we're actually changing then, is that we're no longer using `cy.get(header).should('be.visible');` which is proving very flaky during CI, but is OK during normal browser usage.